data/mods/More_Survival_Tools/start_locations.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
Needs moar !!SCIENCE!! on whether forest_thick is really any more innawoods than the default for this.